docs/resources/rest_managed.md

```diff
@@ -107,6 +107,10 @@ An existing object can be [imported](https://www.terraform.io/docs/import/index.
 
 ```
 terraform import aci_rest_managed.example_tenant uni/tn-{name}
+
+or
+
+terraform import aci_rest_managed.example_tenant `{ "parentDn": "uni/tn-{name}" }`
 ```
```

Comment:
```suggestion

\```
terraform import aci_rest_managed.example_tenant uni/tn-{name}
\```
To support more complex scenario, it can also be imported using a JSON representation using the following command:
\```
terraform import aci_rest_managed.example_tenant `{ "parentDn": "uni/tn-{name}" }`
\```
```